tests/test_marathon_tools.py

```diff
@@ -1030,7 +1030,7 @@ def test_get_constraints_in_config(self):
             service='fake_name',
```

Comment:
can you rename line 1027 to "test_get_constraints_in_config_overrides_other_constraints" or something like that?